app/models/company.rb

```diff
@@ -1,12 +1,24 @@
 require_relative 'institution'
 class Company < Institution
+  attr_accessor :searched_competences
+
   property :size, String
   has n, :segments, :through => Resource
   has n, :company_types, :through => Resource
   has n, :company_areas, :through => Resource
   has n, :competences, :through => :competece_institutions
   has n, :competence_institutions
 
+  def featured_competences
+    if @searched_competences and @searched_competences.length > 0
```

Comment:
Não to ligado nos ruby, mas será que não existe um `empty?` pra verificar se uma coleção está vazia? Daria pra usar no lugar do `.length > 0`.